src/types.mligo

```diff
@@ -166,6 +160,7 @@ type storage =
   ; extra : contract_extra
   ; proposals : (proposal_key, proposal) big_map
   ; proposal_key_list_sort_by_level : (blocks * proposal_key) set
+  ; voters : (address * proposal_key, vote_status) big_map
```

Comment:
@sras I've made the changes according to your feedback, this should be good now.

Reply:
Minor point, but wouldn't this field be better called `votes`, since it track all kinds of vote info (not just who voted)?

Reply:
I've updated this to `staked_votes` now.